src/postprocess/chi0.jl

```diff
@@ -83,10 +77,11 @@ function compute_χ0(ham; droptol=0, temperature=ham.basis.model.temperature)
             ddiff = Smearing.occupation_divided_difference
             ratio = filled_occ * ddiff(model.smearing, E[m], E[n], εF, temperature)
             (n != m) && (abs(ratio) < droptol) && continue
-            # dVol because inner products have a dVol so that |f> becomes <dVol f|
```

Comment:
can you keep that for stupid chemists like me?

Reply:
If you just add the |f> and the <dVol f| in your comment one does not need to know what a "dual" is.